src/bundle/Resources/public/js/scripts/admin.notifications.js

```diff
@@ -0,0 +1,34 @@
+(function (global, doc, eZ, $) {
+    const notificationsContainer = doc.querySelector('.ez-notifications-container');
+    const notifications = JSON.parse(notificationsContainer.dataset.notifications);
+    const addNotification = ({ detail }) => {
+        // @TODO: Unify 'error' and 'danger' label in 3.0.
+        const configKey = detail.label === 'danger' ? 'error' : detail.label;
+        const config = eZ.adminUiConfig.notifications[configKey];
+        const timeout = config ? config.timeout : 0;
+        const container = doc.createElement('div');
+        const notification = `
+            <button type="button" class="close" data-dismiss="alert" aria-label="Close">
```

Comment:
known this is probably not big deal, but I am not fan of mixing those things.

Reply:
"me too" :)

Reply:
By _those things_ you mean what?

Reply:
@sunpietro 
HTML in JS files is on that same level for me as JS in PHP files ;)

Reply:
Don't ever look at the React code ;)

Reply:
I am not sure but I think even for that obvious button we should have title attribute

Reply:
Never said I am fan of that in react too :D